streamlit_app.py

```diff
@@ -1,18 +1,73 @@
+import altair as alt
 import matplotlib.pyplot as plt
+import pandas as pd
 import streamlit as st
+from vega_datasets import data
 
+import twitter.tweet_fetcher
 import twitter.word_cloud
 
-import pandas as pd
-import altair as alt
-from vega_datasets import data
-
 DATA_DIR = "data"
+state_map = {
```

Comment:
I think it might be possible to turn state_map in twitter/state_data_aggregator.py to a global var (i.e. rename it to STATE_MAP) and use `from twitter.state_data_aggregator import STATE_MAP` here instead to avoid redundancy.